contracts/payloads/IGP48/PayloadIGP48.sol

```diff
@@ -0,0 +1,946 @@
+                address(0xE85eb0acB7281fAf00810d167C7dE14bB070B480)
+            );
+
+            uint256 allowance = IERC20(GHO_ADDRESS).allowance(
+                address(FLUID_RESERVE),
+                F_GHO_ADDRESS
+            );
+
+            protocols[0] = F_GHO_ADDRESS;
+            tokens[0] = GHO_ADDRESS;
+            amounts[0] = allowance + (73_000 * 1e6);
+        }
+
+        FLUID_RESERVE.approve(protocols, tokens, amounts);
+    }
+
+    /// @notice Action 5: Adjust Reserve allowances on wBTC/stables and cbBTC/stables vaults
+    function action5() internal {
+         address[] memory protocols = new address[](6);
+        address[] memory tokens = new address[](6);
+        uint256[] memory amounts = new uint256[](6);
+
+        address wBTC_USDC_VAULT = getVaultAddress(21);
+        address wBTC_USDT_VAULT = getVaultAddress(22);
+        address cbBTC_USDC_VAULT = getVaultAddress(29);
+        address cbBTC_USDT_VAULT = getVaultAddress(30);
+
+        { // Supply Side wBTC-USDC
+
+            protocols[0] = wBTC_USDC_VAULT;
+            tokens[0] = WBTC_ADDRESS;
+            amounts[0] = 0.0015 * 1e8;
+        }
+
+        { // Supply Side wBTC-USDT
+
+            protocols[1] = wBTC_USDT_VAULT;
+            tokens[1] = WBTC_ADDRESS;
+            amounts[1] = 0.0015 * 1e8;
+        }
+
+        { // Borrow Side wBTC-USDC
+            uint256 allowance = IERC20(USDC_ADDRESS).allowance(
+                address(FLUID_RESERVE), 
+                wBTC_USDC_VAULT
+            );
+
+            protocols[2] = wBTC_USDC_VAULT;
+            tokens[2] = USDC_ADDRESS;
+            amounts[2] = allowance + (26_000 * 1e6);
+        }
+
+        { // Borrow Side wBTC-USDT
+            uint256 allowance = IERC20(USDT_ADDRESS).allowance(
+                address(FLUID_RESERVE), 
+                wBTC_USDT_VAULT
+            );
+
+            protocols[3] = wBTC_USDT_VAULT;
+            tokens[3] = USDT_ADDRESS;
+            amounts[3] = allowance + (26_000 * 1e6);
+        }
+
+        { // Borrow Side cbBTC-USDC
+            uint256 allowance = IERC20(USDC_ADDRESS).allowance(
+                address(FLUID_RESERVE), 
+                cbBTC_USDC_VAULT
+            );
+
+            protocols[4] = cbBTC_USDC_VAULT;
+            tokens[4] = USDC_ADDRESS;
+            amounts[4] = allowance + (26_000 * 1e6);
+        }
+
+        { // Borrow Side cbBTC-USDT
+            uint256 allowance = IERC20(USDT_ADDRESS).allowance(
+                address(FLUID_RESERVE), 
+                cbBTC_USDT_VAULT
+            );
+
+            protocols[5] = cbBTC_USDT_VAULT;
+            tokens[5] = USDT_ADDRESS;
+            amounts[5] = allowance + (26_000 * 1e6);
+        }
+
+        FLUID_RESERVE.approve(protocols, tokens, amounts);
+    }
+
+    /// @notice Action 6: Add Team Multisig as auth for newly deployed Vaults
+    function action6() internal {
+        VAULT_FACTORY.setVaultAuth(getVaultAddress(63), TEAM_MULTISIG, true);
```

Comment:
GHO-USDC T4 vault is at id 61, which is the last one. so we need to start at 62 here